source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1217,6 +1218,19 @@ message TransformationRecordSynonymousConstants {
 
 }
 
+message TransformationReplaceCopyMemoryWithLoadStore {
+
+   // A transformation that replaces instructions OpCopyMemory with loading
+   // the source variable to an intermediate value and storing this value into the
+   // target variable of the original OpCopyMemory instruction.
+
+   // The intermediate value
+   uint32 source_value = 1;
```

Comment:
For transformations with a single fresh id, I tend to name the field "fresh_id".  Having "fresh" in the name is good for readability.